api/sheet.go

```diff
@@ -37,6 +37,7 @@ const (
 )
 
 // SheetVCSPayload is the additional data payload of the VCS sheet.
+// The sheet source should be one of SheetFromGitLabSelfHost and SheetFromGitHubCom.
 type SheetVCSPayload struct {
```

Comment:
The VCS sheets already have the VCS info in payload. If we need to store tab info into backend, maybe we need to add a `tab` field into SheetVCSPayload.